### PR TITLE
Update the jsdoc comments to modern syntax - Part 5

### PR DESCRIPTION
**Pull request by BrandonOCasey**
## Description
Updated all of tracks and some of utils!

## Requirements Checklist
- [x] Feature implemented / Bug fixed
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: src/js/tracks/track.js [hunk 1]
```diff
@@ -7,15 +7,37 @@ import * as Guid from '../utils/guid.js';
 import EventTarget from '../event-target';
 
 /**
- * setup the common parts of an audio, video, or text track
- * @link https://html.spec.whatwg.org/multipage/embedded-content.html
+ * A Track class taht contains all of the common functionality for {@link AudioTrack},
```
Comment (misteroneill) [inline src/js/tracks/track.js]: Typo - "taht"

File: src/js/utils/guid.js [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
+/**
  * Unique ID for an element or function
  * @type {Number}
- * @private
  */
 let _guid = 1;
 
 /**
- * Get the next unique ID
+ * Get the next unique ID by incrementing the global _guid singleton.
```
Comment (misteroneill) [inline src/js/utils/guid.js]: This addition seems unclear to me. It's not really a singleton, nor is it global. Perhaps just "Get a unique, auto-incrementing ID number that has not been returned before." or something along those lines?

File: src/js/utils/to-title-case.js [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
- * Uppercase the first letter of a string
+ * @param {string} string
+ *        String to be uppercased
  *
- * @param  {String} string String to be uppercased
- * @return {String}
- * @private
- * @method toTitleCase
+ * @return {string}
+ *         The string with an uppercassed first letter
```
Comment (misteroneill) [inline src/js/utils/to-title-case.js]: Typo - "uppercassed"

**Comment by BrandonOCasey** [comment]
@misteroneill Made the requested changes

**Pull request merged**